TidepoolServiceKitUI/View Models/PrescriptionReviewViewModel.swift

```diff
@@ -13,31 +13,41 @@ import LoopKitUI
 import HealthKit
 
 class PrescriptionReviewViewModel: ObservableObject {
+    // MARK: Navigation
     var didFinishStep: (() -> Void)
     var didCancel: (() -> Void)?
     
+    // MARK: State
+    @Published var shouldDisplayError = false
+    
+    // MARK: Prescription Information
     var prescription: MockPrescription?
     let prescriptionCodeLength = 6
+
+    // MARK: Date Picker Information
+    let validDateRange = Calendar.current.date(byAdding: .year, value: -130, to: Date())!...Date()
+    let placeholderFieldText = LocalizedString("Select birthdate", comment: "Prompt to select birthdate with picker")
     
     init(finishedStepHandler: @escaping () -> Void = { }) {
         self.didFinishStep = finishedStepHandler
     }
     
     func entryNavigation(success: Bool) {
         if success {
+            shouldDisplayError = false
             didFinishStep()
         } else {
-           // TODO: handle error
+           shouldDisplayError = true
         }
     }
     
-    func validatePrescriptionCode(prescriptionCode: String) -> Bool {
-        return prescriptionCode.count == prescriptionCodeLength
+    func validatePrescriptionCode(_ prescriptionCode: String, _ birthday: Date) -> Bool {
+        return prescriptionCode.count == prescriptionCodeLength && birthday.timeIntervalSinceReferenceDate > 0
```

Comment:
+1 on the Jan 1, 2001 issue (I verified this is a bug).  Did you mean this to be
```suggestion
        return prescriptionCode.count == prescriptionCodeLength && validDateRange.contains(birthday)
```
?